app/views/adventures/edit.html.erb

```diff
@@ -14,8 +14,8 @@
     <% if !view_only && @adventure.editable_by?(current_user) %>
       <%= link_to 'Edit', [:details, @adventure], class: "SlantButton -large" %>
     <% end %>
-
-    <%= link_to 'Play', @adventure, class: "SlantButton -large" %>
+    <%= link_to 'Debug', [@adventure, {debug: true}], class: "SlantButton -large" %>
+    <%= link_to 'Play', [@adventure, {debug: false}], class: "SlantButton -large" %>
```

Comment:
think it'd be good to omit the `debug` URL param here altogether, so the story creator has a nice clean URL to share in the URL bar.